LoopKit.xcodeproj/xcshareddata/xcschemes/Shared watchOS.xcscheme

```diff
@@ -14,9 +14,9 @@
             buildForAnalyzing = "YES">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "43BA7153201E484D0058961E"
-               BuildableName = "LoopKitUI.framework"
-               BlueprintName = "LoopKitUI"
+               BlueprintIdentifier = "A9E6758022713F4700E25293"
+               BuildableName = "LoopKit.framework"
+               BlueprintName = "LoopKit watchOS"
```

Comment:
The pattern in loop is with a dash. I.e. `LoopKit-watchOS`.  Not strictly necessary to change, but if we want to keep a pattern.